subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -307,55 +501,131 @@ static uint32_t ull_adv_iso_start(struct ll_adv_iso *adv_iso,
 		ticks_slot_overhead = 0;
 	}
 
+	/* setup to use ISO create prepare function for first radio event */
+	mfy_lll_prepare.fp = lll_adv_iso_create_prepare;
+
 	/* TODO: Calculate ISO interval */
 	/* iso_interval shall be at least SDU interval,
 	 * or integer multiple of SDU interval for unframed PDUs
 	 */
-	iso_interval_us = adv_iso->sdu_interval;
+	iso_interval_us = adv_iso->lll.sdu_interval;
 
 	ret_cb = TICKER_STATUS_BUSY;
 	ret = ticker_start(TICKER_INSTANCE_ID_CTLR, TICKER_USER_ID_ULL_HIGH,
-			   (TICKER_ID_ADV_ISO_BASE + adv_iso->bis_handle),
+			   (TICKER_ID_ADV_ISO_BASE + adv_iso->lll.handle),
 			   ticks_anchor, 0,
 			   HAL_TICKER_US_TO_TICKS(iso_interval_us),
 			   HAL_TICKER_REMAINDER(iso_interval_us),
 			   TICKER_NULL_LAZY,
-			   (ll_adv_iso->evt.ticks_slot + ticks_slot_overhead),
-			   ticker_cb, ll_adv_iso,
+			   (adv_iso->evt.ticks_slot + ticks_slot_overhead),
+			   ticker_cb, adv_iso,
 			   ull_ticker_status_give, (void *)&ret_cb);
 	ret = ull_ticker_status_take(ret, &ret_cb);
 
-	return 0;
+	return ret;
 }
 
-static inline struct ll_adv_iso *ull_adv_iso_get(uint8_t handle)
+static void mfy_iso_offset_get(void *param)
```

Comment:
What does this function do?
It seems to search for an available empty air slot to place BIG/BIS in? (trying up to 4 times).
If so, consider renaming as "iso_offset_get" could mean many things. And it sounds like a getter function, not something having side-effects.

Reply:
The function enumerates active ticker instance, and get the offset duration to an ISO event's ticker.
`*_offset_get` is naming is used across the controller implementation, like, for connection window offset, auxiliary PDU offset, periodic advertising offset get etc.